vault/resource_database_secret_backend_connection.go

```diff
@@ -1041,6 +1047,9 @@ func setCassandraDatabaseConnectionData(d *schema.ResourceData, prefix string, d
 	if v, ok := d.GetOkExists(prefix + "connect_timeout"); ok {
 		data["connect_timeout"] = v.(int)
 	}
+	if v, ok := d.GetOkExists(prefix + "skip_verification"); ok {
+		data["skip_verification"] = v.(bool)
+	}
```

Comment:
Do we also need to add this field to the read operation as well?

Reply:
Added [here](https://github.com/hashicorp/terraform-provider-vault/pull/2346/commits/d698db1f3946921470c36ca5f56c14ac0d79156f)